HB/builders.elpi

```diff
@@ -160,6 +160,8 @@ postulate-factories ModName (context-item IDT _ TySkel none t\ context-item IDF
   postulate-factory-abbrev TheType Params IDF GRF TheFactory,
   define-factory-operations TheType Params TheFactory GRF,
   log.coq.env.accumulate current "hb.db" (clause _ _ (current-mode (builder-from TheFactory GRF ModName))),
+  if-verbose (coq.say "Adding clause @using!" IDF "to the context"),
+  log.coq.env.accumulate current "hb.db" (clause _ _ (@using! IDF))
```

Comment:
this seems a big like arming a nuke...

Reply:
We are in a builder section and clauses should be erased at the end.
It makes sense everything in the builder sections depends on the factory.

It does not work, for several reasons: one that I cannot understand: `coq.add-const` seems to ignore it.
However if I "catch it" manually, and feed it back using `@using FetchedString => coq.add-const` I get an anomaly... which I can trace back to Coq 8.13.0. I am going to test it under Coq 8.13.1 to see if the issue persists, and if it does, it I will report.

Reply:
Bingo!